bin/local-env/install-wordpress.sh

```diff
@@ -94,6 +94,10 @@ docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm $CONTAINER touch /var/www/h
 echo -e $(status_message "Activating Google Site Kit plugin...")
 docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI plugin activate google-site-kit --quiet
 
+# Set pretty permalinks.
+echo -e $(status_message "Setting permalink structure...")
+docker-compose $DOCKER_COMPOSE_FILE_OPTIONS run --rm -u 33 $CLI rewrite structure '%postname%' --hard
```

Comment:
Curious: why was this needed?

If the tests fail without pretty permalinks that sounds like a bug to me.

Reply:
It's for the predicate function used in `page.waitForResponse()`, otherwise the REST url uses all query parameters which would require more parsing.

I agree that it should work either way, but we need to identify that request specifically and it both seemed easier and likely more common in the real world to use pretty permalinks.

Reply:
"More parsing" meaning using `reqURL.params.get()` instead of `reqURL.pathname`? 🤔 

But yeah I guess it's fine to keep this. It shouldn't bite us in the future.